store/postgres/migrations/2019-05-03-164052_store_entity_count/up.sql

```diff
@@ -0,0 +1,46 @@
+create function store_entity_count()
+returns void
+language plpgsql
+as $function$
+declare
+  deployment   record;
+  entity_count int8;
+begin
+  -- Initialize the entityCount for all subgraphs
+  update subgraphs.entities e
+  set data = data || '{"entityCount": {"data": "0", "type": "BigInt"}}'
+  where e.entity='SubgraphDeployment';
+
+  -- Handle public.entities
+  with entity_counts as (
+    select subgraph, count(*) as count
```

Comment:
Question: How long do we think this `store_entity_count` function will run for on production?

Reply:
26.5s :wink:  (I just tried it out on the prod db)

Reply:
Ok, great, that's more than acceptable!